src/app/transactions/StakingTransactionForm.tsx

```diff
@@ -64,6 +62,9 @@ export function StakingTransactionForm({
   const [decimals, setDecimals] = useState<number>(0);
   const { transaction, setTransaction, setTransactionHash } = useTransaction();
   const [errors, setErrors] = useState("");
+  const [selectedStakingPosition, setSelectedStakingPosition] =
+    useState<StakingPosition | null>(null);
```

Comment:
can be `undefined` instead of `null` so you can have 
`useState<StakingPosition | undefined>()`

But not an issue

Reply:
https://github.com/AdamikHQ/adamik-app/pull/57/commits/f37a477c314f31f1e80ffb770b9bcae40ce7957e